### ISSUE TITLE
Add original CAD files to the repo

### ISSUE THREAD

**Issue by poldim** [Makerr-Studio/CentrifugalDustSeparator-#9]
I think you're using Inventor for creating these files.  I'd like to make a few modifications before printing and working with STLs is painful.  Can you create an export for Fusion using the [Pack and Go feature](https://www.autodesk.com/support/technical/article/caas/sfdcarticles/sfdcarticles/How-to-export-Inventor-assembly-files-and-open-them-in-Fusion-360.html)?

**Comment by Makerr-Studio** [comment]
Yes Inventor it is for now. I am learning fusion so I can make all the parts native fusion in a way the tube dimensions are easy changeable for someone who has a different tube size. 

Next week when I am back home I will make the Pack and Go for you.

**Comment by phcjpp** [comment]
That would be great for me also. I also have AD Fusion and a Prusa printer and I can't get either the STL or bamboo files to load in either. I am in the UK btw so really appreciate the UK versions! Thanks so much for doing this.

**Comment by poo0ooq** [comment]
Ah, I have the same need as well.

**Comment by rgodin974** [comment]
Hello, you did a good job. I'm also interested in Fusion files. Thank you so much.

**Comment by DBT85** [comment]
This would also be nice for me.

**Comment by Makerr-Studio** [comment]
I am working on the native Fusion Files, this is for the long therm the best solution so you can scale them accordingly.

**Comment by DBT85** [comment]
I am wondering how certain things have been drawn, as scaling the impeller is one thing, but I would not want to scale the 3mm o-ring size, even though obviously we would need to scale the diameter of the oring.

**Comment by Makerr-Studio** [comment]
That is exactly why we need make the files available in native fusion so people can adjust the tube size accordingly to their own exact tube size without scaling te rest accordingly. I will focus on the in and outlets first so you can use them a.s.a.p!

**Comment by poldim** [comment]
> That is exactly why we need make the files available in native fusion so people can adjust the tube size accordingly to their own exact tube size without scaling te rest accordingly. I will focus on the in and outlets first so you can use them a.s.a.p!

yea, for sure!! native fusion files would be best!

btw, there are two ways to share files:
- you can [share](https://help.autodesk.com/view/fusion360/ENU/?guid=ASM-SHARE-DESIGN) them as a link which has the benefit of constantly updating as you make tweaks to them but allow users to download them, this doesn't lend itself to revision control, but has the benefit of always being the latest and greatest version that you've made/tweaked
- you can "[export](https://help.autodesk.com/view/fusion360/ENU/?guid=ASM-EXPORT-DESIGN)" the file from fusion and then drop it in the repo, anytime you have major edits that you'd want to publish, you'd need to re-do the export/upload but that way you have rev control

**Comment by Makerr-Studio** [comment]
I am working on the Fusion File for de MK2_EU version of the separator. Today I made the 3D models of the Inlet, Outlet, and Impeller Shroud. So you are able to change the dimensions more easy. I made a Fusion Archive file of the assembly: https://github.com/Makerr-Studio/CentrifugalDustSeparator-/tree/main/MK2_EU/Fusion 

Are you able to open it?
